optuna/multi_objective/trial.py

```diff
@@ -42,14 +42,16 @@ def __init__(self, trial: Trial):
         # See also: https://github.com/optuna/optuna/pull/1054/files#r407982636
         self._n_objectives = multi_objective.study.MultiObjectiveStudy(trial.study).n_objectives
 
-    def suggest_float(self, name: str, low: float, high: float, *, log: bool = False) -> float:
+    def suggest_float(
+            self, name: str, low: float, high: float, *, log: bool = False, step: Optional[float] = None
```

Comment:
@himkt Is it okay?

Reply:
It looks good to me.

Reply:
Thanks! I should have clarified the modification. I have added `step: Optional[float] = None` as argument of `suggest_float`.